### PR TITLE
Write csvs to blob storage

### PR DESCRIPTION
**Pull request by graham-skylight**

### CODE REVIEW

File: src/FunctionApps/python/GenerateCSVs/__init__.py [hunk 1]
```diff
@@ -1,47 +1,54 @@
-import csv
-import io
-
-import azure.functions as func
```
Comment (bamader) [inline src/FunctionApps/python/GenerateCSVs/__init__.py:4]: Why is the git diff having a fit over this 🤷

**Review by bamader — APPROVED** [review]
LGTM. Honestly seems like git + the requirement to have tests is making a lot of overhead out of what amounts to a few very small changes: adding the blob configs in the __init__ function, changing some capitalization, adding a new function which just gets some creds, and modifying a signature for writing purposes. No issues here.

File: src/FunctionApps/python/GenerateCSVs/__init__.py (hunk 1 above)
Comment (graham-skylight) [reply src/FunctionApps/python/GenerateCSVs/__init__.py:4]: `dos2unix` - azure's templates all (surprise!) have windows line endings

**Comment by graham-skylight** [comment]
@bamader Yeah, this one's really not too huge, just a thing I neglected to do in the initial skeleton

**Pull request merged**